test/module/shared_model/validators/commands_validator_test.cpp

```diff
@@ -0,0 +1,316 @@
+/**
+ * Copyright Soramitsu Co., Ltd. 2017 All Rights Reserved.
+ * http://soramitsu.co.jp
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *        http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include <gtest/gtest.h>
+#include "builders/protobuf/transaction.hpp"
+#include "utils/polymorphic_wrapper.hpp"
+
+iroha::protocol::Transaction generateEmptyTransaction() {
```

Comment:
What is the reason to make those methods? Maybe better to use builders instead?

Reply:
IR-701 in Jira for that